### PR TITLE
✨ feat: added news endpoints

### PR DESCRIPTION
**Pull request by mdmuaj13**
Added Tech-based news to the news endpoint.
Following endpoints:
```
- /news/{qty}
- /news/slug/{slug}
```

And news object following 
```
interface INews {
  title: string;
  slug: string;
  author: string;
  published_date: string;
  summary: string;
  language: string;
  category: string;
}
```

### CODE REVIEW

**Comment by mdmuaj13** [comment]
PR for issue #218

File: modules/news/api/news-routes.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import { getAllNews, getNewsBySlug } from '../utils/getNews';
+import { INews } from '../utils/news.interface';
+
+/**
+ * @openapi
+ * definitions:
+ *   MockNews:
+ *     type: array
+ *     items:
+ *       type: object
```
Comment (ageddesi) [inline modules/news/api/news-routes.ts:14]: Hi, can you move this definition to the interface file please. :)

File: modules/news/api/news-routes.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+     *         description: OK
+     *         schema:
+     *           $ref: '#/definitions/MockNews'
+     *             
+     */
+    app.get('/news/:qty', (req: Request, res: Response) => {
+        let qty = getQtyFromRequest(req);
+
+        if (!qty) {
+            qty = 10;
```
Comment (ageddesi) [inline modules/news/api/news-routes.ts:64]: You can provide a default to the getQtyFromRequest function.

File: modules/news/api/news-routes.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        }
+
+        const data : INews[] = getAllNews(qty);
+
+        if (!data || data.length === 0) {
+            return res.status(404).json();
```
Comment (ageddesi) [inline modules/news/api/news-routes.ts:70]: Please can you change this to a 400 error and provide a message in return. We reserve 404 for when a url does not exist.

**Review by ageddesi — CHANGES_REQUESTED** [review]
Thank you for the work on this, I have asked for a few minor changes. 
Also I have not used slugify before so that was a nice library to learn about

**Review by ageddesi — DISMISSED** [review]

**Review by ageddesi — APPROVED** [review]

**Comment by ageddesi** [comment]
Thanks for the work on this @mdmuaj13 Happy Hacktoberfest 👍

**Comment by mdmuaj13** [comment]
Thanks @ageddesi. See you soon with another issue 😎

**Pull request merged**